cognite/client/_api/functions.py

```diff
@@ -768,7 +769,7 @@ def get_response(
             function_external_id (str, optional): External ID of the function on which the call was made.
 
         Returns:
-            Response from the function call.
+            Dict[str, Any] | None: Response from the function call.
```

Comment:
@HMEiding note that the official docs list the `"response" (object)` in the API repsonse as required, but it is actually missing when the function returns `None` (i.e. this returns `None`). https://developer.cognite.com/api/v1/#tag/Function-calls/operation/getFunctionCallResponse